daemon/cluster/services.go

```diff
@@ -266,7 +265,7 @@ func (c *Cluster) CreateService(s swarm.ServiceSpec, encodedAuth string, queryRe
 				// "ctx" could make it impossible to create a service
 				// if the registry is slow or unresponsive.
 				var cancel func()
-				ctx = compatcontext.WithoutCancel(ctx)
+				ctx = context.WithoutCancel(ctx)
```

Comment:
Note that the comment here no longer matches the code. I left the comment as-is for now, because I wasn't sure if this would still require an _actual_ fresh context here, or if `WithoutCancel` is enough https://github.com/moby/moby/blob/52333f3a34b430f2a0800ee2638c1a2b7e41ac82/daemon/cluster/services.go#L262-L270



Note that there's another place where the same comment was added, and also would need updating.

- "use a fresh context" comment was added in https://github.com/moby/moby/commit/f8273a216ed35e22ac157dee8055393f07d4be39
- This commit kept the context, but instead added `WithoutCancel`: https://github.com/moby/moby/commit/677d41aa3b1146b0dfc8f5889bd3bd3fa9c128b2 
- This refactor removed the "Cluster.getRequestContext()" wrapper: https://github.com/moby/moby/commit/83c532c9b7e39977e7f61e0e837f0bae394d31e1